src/openapi/peridio-admin-openapi.yaml

```diff
@@ -124,6 +124,40 @@ paths:
                 properties:
                   data:
                     $ref: "#/components/schemas/ca-certificate"
+    put:
+      summary: update ca certificate
+      tags:
+        - "CA Certificates"
+      parameters:
+        - name: organization_name
+          in: path
+          required: true
+          schema:
+            $ref: "#/components/schemas/organization-name"
+        - name: ca_certificate_serial
+          in: path
+          required: true
+          schema:
+            $ref: "#/components/schemas/ca-certificate-serial"
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              properties:
+                description:
+                  type: string
+                jitp:
```

Comment:
<img width="451" alt="image" src="https://user-images.githubusercontent.com/868014/194961959-cdd5c240-a4fc-422e-bf11-7bb26bc892a1.png">
Is it possible to get it to say required in red only under the 3 subfields of JITP, but continue to not say required on JITP itself?